addenda02_test.go

```diff
@@ -38,14 +38,10 @@ func TestMockAddenda02(t *testing.T) {
 func testAddenda02ValidRecordType(t testing.TB) {
 	addenda02 := mockAddenda02()
 	addenda02.recordType = "63"
-	if err := addenda02.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda02.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
recordType doesn't seem correct. 

I think the following code should return the fieldError

	if addenda02.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda02.recordType)
	}

Reply:
In general it seems the validation for recordType do not return the error.   The validation did at one point in time

for example:

	if bh.recordType != "5" {
		msg := fmt.Sprintf(msgRecordType, 5)
		return &FieldError{FieldName: "recordType", Value: bh.recordType, Msg: msg}
	}

If it is agreeable, create an issue and update the code (code base) to return the error.

Reply:
This PR is already pretty large, and to make it easier to review I don't want to add logic changes. So I created an issue https://github.com/moov-io/ach/issues/469, and I plan to work on making the changes you've suggested here to fix this tomorrow.

Reply:
That sounds great.  Thank you!